### DISCUSSION TITLE
Filter for Zone Records

### DISCUSSION THREAD

**Discussion by dkebler** [TechnitiumSoftware/DnsServer#923] [Ideas]
Anyone ever asked to be able to filter records when editing a zone.   Route53 has this and it really helps to find and edit specific records when you have dozens of them.   Could filter and any part of the record by  name,  the comment field and the record type.  

If no I'd like to make a "issue" to request that

just want to add this this project is the BEST full DNS server out there and the browser UI is a big part of that.   I'm really happy to dump dnsmasq, unbound, and other full DNS servers which I tried and found to complicated or lacking.

**Comment by ShreyasZare** [comment]
Thanks for the request and compliments. The record filtering is already [planned](https://github.com/TechnitiumSoftware/DnsServer/issues/301#issuecomment-1788546660) and will be implemented as soon as feasible. This will allow filtering records similar to how there are options to filter the Query Logs in the Logs section.
